builtin/logical/transit/path_keys.go

```diff
@@ -238,7 +238,7 @@ func (b *backend) pathPolicyRead(ctx context.Context, req *logical.Request, d *f
 			"supports_decryption":    p.Type.DecryptionSupported(),
 			"supports_signing":       p.Type.SigningSupported(),
 			"supports_derivation":    p.Type.DerivationSupported(),
-			"auto_rotate_interval":   p.AutoRotateInterval.String(),
+			"auto_rotate_interval":   int64(p.AutoRotateInterval.Seconds()),
```

Comment:
Are we set on "_interval"? If we use "_period" then the CLI will format the output into a human-friendly timestamp.

Reply:
Nope, no attachments from me! That suffix will likely help a decent amount in this case by removing a couple orders of magnitude; most of the values for this will likely be on the order of days, weeks, or months.

Reply:
I'll submit a separate PR with the renaming of this field to keep this changeset small.